src/plugins/vis_type_timeline/server/handlers/chain_runner.js

```diff
@@ -47,7 +47,9 @@ export default function chainRunner(tlConfig) {
   let sheet;
 
   function throwWithCell(cell, exception) {
-    throw new Error(' in cell #' + (cell + 1) + ': ' + exception.message);
+    const e = new Error(exception.message);
+    e.name = `Expression parse error in cell #${cell + 1}`;
```

Comment:
does the name field matter?

Reply:
Yes: server-side error messages are propagated to the client and the toast error message will show. This is needed to make the toast more readable (stack information is kept hidden from client)